.github/workflows/trigger-cc-action.yaml

```diff
@@ -0,0 +1,26 @@
+name: trigger-cc-action
+on:
+  push:
+    paths:
+      - /data/wip/*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - id: files
+      uses: jitterbit/get-changed-files@v1
```

Comment:
What is this doing? If nothing changed exit?

Reply:
Thanks!!